### PR TITLE
feat: updated the welcome page

### PR DESCRIPTION
**Pull request by hitohata**
## Overview

updated the welcome page in the project document.

### CODE REVIEW

File: project-document/docs/intro.md [hunk 1]
```diff
@@ -3,3 +3,31 @@ slug: /
...
 # Welcome to Nishiki Project Document
+
+This is Nishiki's document.
+Nihiki is developing for saving your food.
+
+Do you know what is contained in your refrigerator correctly?
+I have experienced that I bought some food that I had already bought, or I missed buying food that I had thought those remained in the refrigerator.
+
+Using Nishiki, you don't need to worry about it!
+Nishiki manages what foods are in the refrigerator for you and your partner.
```
Comment (yukiyohure) [inline project-document/docs/intro.md:14]:
```
This is Nishiki's document.
Nishiki is a solution for food management.

Do you ever find yourself unsure about what's in your fridge?
Perhaps you've bought groceries you already had or forgot to buy ones you thought were still in the fridge.

With Nishiki, you no longer need to worry about this!
Nishiki manages the inventory of your fridge for both you and your partner.
```

File: project-document/docs/intro.md [hunk 2]
```diff
@@ -3,3 +3,31 @@ slug: /
...
+Using Nishiki, you don't need to worry about it!
+Nishiki manages what foods are in the refrigerator for you and your partner. 
+
+## Features
+
+### Manage your refrigerator
+
+Nishiki manages your refrigerator for you.
+But not only for you, you can share it with your partner, friends, and so on.
+You no longer need to ask your partner what food remains in your refrigerator or worry about duplicates of the same food when you have a party.
```
Comment (yukiyohure) [inline project-document/docs/intro.md:22]:
```
Nishiki not only manages your fridge for you, but it also enables you to share it with your partner, friends, and more.
You'll no longer need to ask your partner about the food in your fridge or worry about having duplicate food when you buy groceries for a party.
```

**Review by yukiyohure — COMMENTED** [review]
I left ideas for revision in the comments.
I'd appreciate it if you could check it and update it if it looks good 👍

**Comment by hitohata** [comment]
@yukiyohure 
Thank you for review.
I reflected on your review comment.

**Review by yukiyohure — APPROVED** [review]
Mbappé LGTM

**Pull request merged**